doc/source/getting_started/running_mapdl.rst

```diff
@@ -138,7 +132,25 @@ Alternatively you can use a hostname:
 
 Note that you must have started MAPDL in gRPC mode on the computer with
 the mentioned IP address/hostname for this to work.
+
 If you have MAPDL installed on your local host, you
 can use the ``launch_mapdl`` method to both start and connect to MAPDL.
 
 If you have any problem launching PyMAPDL, see :ref:`debugging_launch_mapdl`
+
+Connect to a dockerized MAPDL session
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you work with Docker, for example using the
+`docker-compose.yml <pymapdl_docker_compose_base_>`_ file provided in the
+directory `docker <pymapdl_docker_dir_>`_, then you need to connect to the
+MAPDL instance running on the docker container as if it was a remote MAPDL
+instance. However, because of the ports are exposed to the host (``ports`` field
+in `docker-compose.yml <pymapdl_docker_compose_base_>`_ file) you can connect
+to it without specifying the IP address of the Docker container.
```

Comment:
```suggestion
If you work with Docker, for example you use the
`docker-compose.yml <pymapdl_docker_compose_base_>`_ file provided in the
directory `docker <pymapdl_docker_dir_>`_, then you must connect to the
MAPDL instance running on the Docker container as if it was a remote MAPDL
instance. However, because the ports are exposed to the host via the ``ports`` field
in the `docker-compose.yml <pymapdl_docker_compose_base_>`_ file, you can connect
to this MAPDL instance without specifying the IP address of the Docker container.
```